staging/2024/README.md

```diff
@@ -0,0 +1,29 @@
+
+## Metadata Files for Generating euc_county_zipcode_crosswalk.csv
+
+To generate the `euc_county_zipcode_crosswalk.csv` file, you'll need the following metadata files:
+
+- **euc_counties.csv: County Names and States**
+  Create a file manually by extracting county names and states from the EUC
+  policy. Refer to the
+  [2024-MIPS-Automatic-EUC-Policy-Clean.docx](https://github.com/CMSgov/qpp-eu-data/tree/main/staging/2024/2024-MIPS-Automatic-EUC-Policy-Clean.docx)
+  for the 2024 Merit-based Incentive Payment System Automatic Extreme and
+  Uncontrollable Circumstances Policy.
+
+  > [!NOTE]
+  > This file is normally a `.pdf` file downloaded from the qpp documents page, but
+  this year it was provided as a `.docx` emailed to Shane Dougherty.
```

Comment:
```suggestion
  > this year it was provided as a `.docx` emailed to Shane Dougherty.
```

Reply:
Also wondering if we need to version control specific people? Maybe just "emailed to the QPP Eligibility team's product owner" (or whatever the actual title is)

Reply:
I think in this case leaving Shane might actually be best, the record is in his specific inbox.  Not a generic product owner inbox.   I just more wanted to explain why it is different this year.